tests/ffi/selinux/test.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# shellcheck disable=SC1091
+. ../common/prepare.sh
+
+disk_cleanup
+prepare_test
+reload_config
+
```

Comment:
I think you should call 
run_container_in_qm() it starts the container inside qm, later on  you are using podman exec on that container

lets see the test will fail in gihub actions

Reply:
I'm just getting the error message from the qm, I don't need to start any containers inside.
Its a really simple test. 
Just running the `podman exec -it qm setenforce 0` and trying to get back `setenforce:  security_setenforce() failed:  Permission denied`

Reply:
Ahh got it, thanks

Reply:
so I think there is no need to call these functions here, right?

Reply:
Removed them.